src/components/RangeSlider/components/SingleThumb/SingleThumb.scss

```diff
@@ -182,6 +182,20 @@ $range-thumb-shadow-focus: var(
       background: var(--p-border-disabled, color('white'));
     }
   }
+
+  &.newDesignLanguage {
+    @include range-thumb-selectors {
+      box-shadow: 0 0 0 0 var(--p-focused);
+      transition: box-shadow duration(fast) var(--p-ease);
+    }
+
+    &:focus {
+      @include range-thumb-selectors {
+        box-shadow: 0 0 0 rem(1px) var(--p-surface),
+          0 0 0 rem(3px) var(--p-focused);
+      }
+    }
```

Comment:
Had to do a slightly hacky iteration for single slider. The reason being that we can't use `position:absolute` or `:after` on the slider element.